### PR TITLE
BD-2677 Add information on segmentation filtering

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #BD-2677

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Custom attribute objects can contain [data types][1], such as:
 - Booleans
 - Arrays
 - Time
+  - When filtering a custom attribute object with a Time data type, you can choose to filter based on "Day of Year" or "Time". "Day of Year" will check only the month and day for comparison. "Time" will compare the full timestamp, including the year.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md:33]: There's no need to impose styling like capitalization on the data types. Also, a custom attribute "object", I'm not sure if adding "object" here is accurate.

```suggestion
  - When filtering a nested time custom attribute, you can choose to filter based on "Day of Year" or "Time". "Day of Year" will check only the month and day for comparison. "Time" will compare the full timestamp, including the year.
```

Comment (bre-fitzgerald) [reply _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md:33]: You can get an idea for how we talk about data types and attributes in the main [Custom attributes](https://www.braze.com/docs/user_guide/data_and_analytics/custom_data/custom_attributes/) article, for consistency. And then whatever wording you land on here, if you could also update the others ones 🙏

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Looks good, just one comment about how we stylize and refer to data types 👀

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md [hunk 2]
```diff
@@ -287,6 +288,12 @@ When working with nested custom attributes segmentation, you'll have access to a
 
 ![A user choosing an operator based on the data type for the nested custom attribute][7]
 
+### Filtering for Time data types
+
+When filtering a custom attribute object with a Time data type, you can choose to filter with operators under the **Day of Year** or **Time** categories when comparing the date value.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes/nested_custom_attribute_support.md:293]:
```suggestion
When filtering a nested time custom attribute, you can choose to filter with operators under the **Day of Year** or **Time** categories when comparing the date value. 
```

File: _docs/_user_guide/engagement_tools/segments/segmentation_filters.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ glossaries:
     tags:
       - Custom attributes
   - name: Nested Custom Attributes
-    description: Attributes that are the properties of custom attributes.
+    description: Attributes that are the properties of custom attributes.<br><br>When filtering a custom attribute object with a Time data type, you can choose to filter based on "Day of Year" or "Time". "Day of Year" will check only the month and day for comparison. "Time" will compare the full timestamp, including the year.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/segments/segmentation_filters.md:54]:
```suggestion
    description: Attributes that are the properties of custom attributes.<br><br>When filtering a nested time custom attribute, you can choose to filter based on "Day of Year" or "Time". "Day of Year" will check only the month and day for comparison. "Time" will compare the full timestamp, including the year.
```

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**